index.ts

```diff
@@ -80,3 +84,10 @@ export { getChecksumAddress } from './src/core/get-checksum-address.js';
  * @since 0.1.1
  */
 export * as IPFS from './src/ipfs.js';
+
+/**
+ * This module provides high-level utility functions for working with the Knowledge Graph API.
+ *
+ * @since 0.5.0
+ */
+export * as Graph from './src/graph/index.js';
```

Comment:
Hmm yeah not sure what we call this namespace. `Graph` seems fine I suppose.

Reply:
Yeah, no better idea here so far